### ISSUE TITLE
Comment Management

### ISSUE THREAD

**Issue by bahadirtortop** [andrewsegas/instaboost#3]
You should comment management on User Interface.

**Comment by andrewsegas** [comment]
Hi, i don't know if this commit can help  you, but here b0b9983996788715a9e99a7c584aa1f83d8eb74f comments is working by: example.py or instaboost.py (if you update de tag comments_list)
otherwise in file .exe it still in maintenance

**Comment by bahadirtortop** [comment]
Yeah I updated both instaboot and example file. I never gonna changed. Can
you check it and send us screenshot please.

2018-03-16 3:03 GMT+03:00 Andrews Fernandes Egas <notifications@github.com>:

**Comment by andrewsegas** [comment]
You need to update everything, instabot.py, example.py, userinfo.py...
